src/ugeneui/src/app_settings/user_apps_settings/UserApplicationsSettingsGUIController.cpp

```diff
@@ -111,7 +113,17 @@ UserApplicationsSettingsPageWidget::UserApplicationsSettingsPageWidget(UserAppli
         it.next();
         langCombo->addItem(it.key(), it.value());
     }
-    styleCombo->addItems(QStyleFactory::keys());
+    auto keys = QStyleFactory::keys();
```

Comment:
nit: the 10 lines below can be replaced with only 3:
```
for () {
  styleCombo->addItem(map.value(key, key));
}
```

Reply:
Done